src/Nethermind/Nethermind.Runner/configs/holesky.cfg

```diff
@@ -0,0 +1,37 @@
+{
+    "Init": {
+        "ChainSpecPath": "chainspec/holesky.json",
+        "GenesisHash": "0xff9006519a8ce843ac9c28549d24211420b546e12ce2d170c77a8cca7964f23d",
+        "BaseDbPath": "nethermind_db/holesky",
+        "LogFileName": "holesky.logs.txt"
+    },
+    "Sync": {
+        "FastSync": true,
+        "SnapSync": true
```

Comment:
Wondering if it is safe to start completly new chain with SnapSync=true? In case Geth won't work for some reason there will be noone to serve SnapData so we would be locked?wdyt?
@MarekM25

Reply:
Agree. removed snap for now

Reply:
Just wondering now only about bootnodes -I guess it would be Geth, right?

Reply:
Okay, we can switch it later :) Both options are reasonable IMO, but let's not forget later to enable it